### PR TITLE
Inspect image test

### PR DESCRIPTION
**Pull request by orius123**

### CODE REVIEW

File: content/io/cloudslang/docker/images/inspect_image.sl [hunk 1]
```diff
@@ -20,43 +20,68 @@
...
     - host
     - port:
-        default: "'22'"
+        required: false
     - username
-    - password
+    - password:
+        required: false
     - privateKeyFile:
-        default: "''"
```
Comment (Bonczidai) [inline content/io/cloudslang/docker/images/inspect_image.sl]: do we want to use snake case for all flow inputs or is it ok for now?

File: test/io/cloudslang/docker/images/inspect_image.inputs.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+####################################################
+test_docker_inspect_image_success:
+  testSuites: [docker,aaa]
+  description: Tests that inspect image flow finishes with success
```
Comment (Bonczidai) [inline test/io/cloudslang/docker/images/inspect_image.inputs.yaml]: `aaa` ?

File: test/io/cloudslang/docker/images/test_inspect_image.sl [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    - inspect_image:
+        do:
+          images.inspect_image:
+            - host
+            - port
+            - username
+            - password
+            - image_name
+        publish:
+            - standard_out: standard_out
```
Comment (Bonczidai) [inline test/io/cloudslang/docker/images/test_inspect_image.sl]: why not just `- standard_out` ?

File: test/io/cloudslang/docker/images/test_inspect_image.sl [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+          FAILURE: FAILURE
+
+    - verify_output:
+        do:
+          strings.string_occurrence_counter:
+            - string_in_which_to_search: standard_out
+            - string_to_find: "'/hello'"
+        navigate:
+          SUCCESS: clear_after
+          FAILURE: VERIFY_FAILURE
```
Comment (Bonczidai) [inline test/io/cloudslang/docker/images/test_inspect_image.sl]: shouldn't you check the number of occurrences in another task ?

**Comment by Bonczidai** [comment]
:+1:

File: content/io/cloudslang/docker/images/inspect_image.sl (hunk 1 above)
Comment (orius123) [inline content/io/cloudslang/docker/images/inspect_image.sl]: My bad. Should be required false

File: test/io/cloudslang/docker/images/inspect_image.inputs.yaml (hunk 2 above)
Comment (orius123) [inline test/io/cloudslang/docker/images/inspect_image.inputs.yaml]: Oops..

File: test/io/cloudslang/docker/images/test_inspect_image.sl (hunk 3 above)
Comment (orius123) [inline test/io/cloudslang/docker/images/test_inspect_image.sl]: Right

File: test/io/cloudslang/docker/images/test_inspect_image.sl (hunk 4 above)
Comment (orius123) [inline test/io/cloudslang/docker/images/test_inspect_image.sl]: Occurnces of the keyword?

Comment (Bonczidai) [inline test/io/cloudslang/docker/images/test_inspect_image.sl]: you're right.. just looked in the op - SUCCESS means `string is found at least once` :wink:

**Pull request merged**